apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -52,19 +52,24 @@ export default class ChangeAction extends BaseRushAction {
         '',
         'The possible types of changes are: ',
         '',
-        'MAJOR - these are breaking changes that are not backwards compatible. ' +
```

Comment:
What are those extra spaces for?

Reply:
It seems that my auto-linting reformatted the whitespace throughout this file. I will turn it off and revert the changes.